jdaviz/configs/cubeviz/plugins/moment_maps/tests/test_moment_maps.py

```diff
@@ -47,9 +47,9 @@ def test_moment_calculation(cubeviz_helper, spectrum1d_cube, tmpdir):
     label_mouseover = cubeviz_helper.app.session.application._tools['g-coords-info']
     label_mouseover._viewer_mouse_event(flux_viewer, {'event': 'mousemove',
                                                       'domain': {'x': 0, 'y': 0}})
-    assert flux_viewer.state.slices == (0, 0, 1)
+    assert flux_viewer.state.slices == (0, 0, 0)
```

Comment:
This is cause by glue-core v1.9.0 . I tried to bisect glue-core and git blamed https://github.com/glue-viz/glue/commit/da68d1a63adfd6f91aac1e7d2cf864a359c33baa though my bisect-fu isn't strong so we need @astrofrog to explain how that commit could have changed the `viewer.state.slices` default behavior here.

Reply:
Maybe we should go with #2136 instead if this behavior change is undesirable.